tests/sandbox/linalg/test_linalg.py

```diff
@@ -152,3 +152,27 @@ def test_matrix_inverse_solve():
     node = matrix_inverse(A).dot(b).owner
     [out] = inv_as_solve.transform(None, node)
     assert isinstance(out.owner.op, Solve)
+
+
+def test_cholesky_dot_lower():
+    cholesky_lower = Cholesky(lower=True)
+
+    L = matrix("L")
+    L.tag.lower_triangular = True
+
+    C = cholesky_lower(L.dot(L.T))
+    f = pytensor.function([L], C)
```

Comment:
Since we already have the function, can you add a dumb test that `f(Lv.T)` is the same as `cholesky(Lv.dot(Lv.T))` where Lv is an appropriate test value for L. Some for the other test

Reply:
Let me check my understanding. I'm assuming you meant `f(Lv)` not ` f(Lv.T)`, otherwise I'm not sure that I understand? If that's what you mean, then do you want to test some concrete value through `f`, say

```
Lv = np.array([[2,0],[1,4]])
assert np.all(
    np.isclose(
        scipy.linalg.cholesky(np.dot(Lv, Lv.T), lower=True),
        f(Lv),
    ))
```

Reply:
Yes that's what I meant. Apologies if I mistook the formula above :)